client/extensions/woocommerce/app/settings/payments/payments-off-site.js

```diff
@@ -7,13 +7,40 @@ import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-import Card from 'components/card';
+
 import ExtendedHeader from '../../../components/extended-header';
+import ListHeader from '../../../components/list/list-header';
+import ListRows from '../../../components/list/list-rows';
+import ListTable from '../../../components/list/list-table';
+import ListRowField from '../../../components/list/list-row-field';
+import PaymentMethodRow from './payment-method-row';
 
 class SettingsPaymentsOffSite extends Component {
+	constructor( props ) {
+		super( props );
+
+		//TODO: use redux state and real data
+		this.state = {
```

Comment:
Minor since this is temporary, but since props are not needed to set the state, the constructor could be dropped.

[Example](https://github.com/Automattic/wp-calypso/commit/f6982493ca56823d49609e1be9ee32d1acae78c7#diff-de58a3db9f3b73767772821d720b47a6R22).